ui/App/TransactionCenter.svelte

```diff
@@ -9,10 +10,23 @@
   import Center from "../DesignSystem/Component/Transaction/Center.svelte";
 
   const select = event => push(path.transactions(event.detail));
+
+  $: centerIn = { delay: 240, duration: 320, x: 500 };
 </script>
 
+<style>
+  .center {
+    bottom: 32px;
+    position: fixed;
+    right: 32px;
+    z-index: 900;
+  }
+</style>
+
 <Remote {store} let:data={transactions}>
   {#if transactions.length > 0}
-    <Center on:select={select} summary={$summary} {transactions} />
+    <div class="center" in:fly={centerIn}>
```

Comment:
You can inline the params like so:
```suggestion
    <div class="center" in:fly={{ delay: 240, duration: 320, x: 500 }}>
```

then you don't need the ` $: centerIn = { delay: 240, duration: 320, x: 500 };` above.

Reply:
The inlining doesn't really help with readability, having them in assignments also lends it well to be extracted into  library code.

Reply:
In that case the `$:` reactive statement is not necessary.